### PR TITLE
Documentation: Update change log and project metadata

### PR DESCRIPTION
**Pull request by amotl**
## About
This patch mostly updates the change log to include all the latest improvements. Thank you so much for your contributions, @intermittentnrg and @maurerle. 💯

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@
 .. image:: https://img.shields.io/pypi/pyversions/grafanimate.svg
     :target: https://pypi.org/project/grafanimate/
 
-.. image:: https://img.shields.io/badge/Grafana-5.x%20--%208.x-blue.svg
+.. image:: https://img.shields.io/badge/Grafana-9.x%20--%2011.x-blue.svg
```
Comment (amotl) [inline README.rst:17]: Is it correct like this?

Comment (maurerle) [reply README.rst:17]: As a lot of this uses the scene context, we require to have the scene api enabled.
This was hidden behind a feature switch from grafana 10-11.3 and is enabled by default from 11.3

So I would at least bump this to 10, but I think 11.3 makes more sense?

File: CHANGES.rst [hunk 2]
```diff
@@ -5,8 +5,23 @@ grafanimate changelog
...
+- OCI: Added Dockerfile, and started publishing OCI images to GHCR
+- OCI: Dockerfile: Added ``xauth`` and ``ffmpeg`` packages
+- OCI: Dockerfile: Use ``tini`` for bootstrapping ``xvfb-run``
+- Improved finding executable to ``firefox-esr``. Thanks, @gogglespisano.
+- Improved support with ``AnimationSequence.recurrence``
+- Migrated to ``pyproject.toml``
+- Added support for Python up to version 3.13, and dropped support for
+  Python up to version 3.8.
+- Added support for recent versions of Grafana based on React.js
+- Grafana Studio: Update title in solo panel mode, improve login method,
```
Comment (maurerle) [inline CHANGES.rst:17]: We also dropped support for grafana without scenes feature

**Review by maurerle — APPROVED** [review]

File: CHANGES.rst (hunk 2 above)
Comment (amotl) [reply CHANGES.rst:17]: Thanks. I've added a relevant item to the change log.

**Pull request merged**